### PR TITLE
ISSD-1489 [Marketplace] Fixed behavior to update a user, if he has multiple account

### PR DESCRIPTION
**Pull request by welitonpe**
**[ISSD-971](https://liferay.atlassian.net/browse/ISSD-971)**

### CODE REVIEW

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedSolutions/hooks/useHandleAccount.ts [hunk 1]
```diff
@@ -99,7 +99,7 @@ const useHandleAccount = ({
 			type: accountTypes.PERSON,
 		};
 
-		if (form.accountQuantity === 1 || form.accountQuantity > 1) {
+		if (form.accountQuantity === 1 || form.accountQuantity >= 1) {
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedSolutions/hooks/useHandleAccount.ts:102]:
```suggestion
		if (form.accountQuantity >= 1) {
```

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Comment by welitonpe** [comment]
c:test:sf

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**